src/Effects.js

```diff
@@ -158,6 +163,77 @@ module.exports = class Effects {
       },
     };
 
+    this.higher_power = {
+      init: function () {},
+      draw: function () {
+        const numSymbols = 19;
+        // The symbols are arranged to roughly fill a circle.
+        const offsets = [
+          [0, 270],
+          [61, 112],
+          [164, 1],
+          [312, 1],
+          [428, 66],
+          [591, 153],
+          [603, 323],
+          [491, 502],
+          [369, 502],
+          [237, 542],
+          [61, 438],
+          [200, 134],
+          [341, 173],
+          [473, 215],
+          [483, 340],
+          [353, 333],
+          [193, 419],
+          [141, 255],
+          [242, 248]
+        ];
+
+        let ctx = p5._renderer.drawingContext;
+
+        ctx.save();
+        let gradient = ctx.createLinearGradient(425, 425, 425, 0);
+        gradient.addColorStop(1, constants.HIGHER_POWER_COLORS[getCurrentPalette()][0]);
+        gradient.addColorStop(0, constants.HIGHER_POWER_COLORS[getCurrentPalette()][1]);
+        ctx.fillStyle = gradient;
+        ctx.fillRect(0, 0, 425, 425);
+        ctx.restore();
```

Comment:
out of curiosity: what does `save()` and `restore()` do?

Reply:
I just borrowed this from similar code in the same file, but it turns out that this [saves](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/save) and then restores the state of the regular HTML5 [canvas](https://p5js.org/reference/#/p5/drawingContext).  I presume that in particular it restores the original `fillStyle`.